packages/vm/src/params.ts

```diff
@@ -67,10 +68,8 @@ export const paramsVM: ParamsDict = {
    * Ethereum state using a unified verkle tree (experimental)
    */
   6800: {
-    // kaustinen 6 current uses this address, however this will be updated to correct address
-    // in next iteration
     // config
-    historyStorageAddress: '0xfffffffffffffffffffffffffffffffffffffffe', // The address where the historical blockhashes are stored
+    historyStorageAddress: '0x0aae40965e6800cd9b1f4b05ff21581047e3f91e', // The address where the historical blockhashes are stored
```

Comment:
Why doesn't this match the EIP 2935 address?

Reply:
Left is as is but adjusted the eip 2935 one from evm